app/helpers/spree/api/api_helpers.rb

```diff
@@ -0,0 +1,112 @@
+module Spree
+  module Api
+    module ApiHelpers
+      def required_fields_for(model)
```

Comment:
This really looks like a duplicate of Rails 4's [strong parameters](https://github.com/rails/strong_parameters) :see_no_evil: